src/corehost/cli/comhost/clsidmap.cpp

```diff
@@ -79,9 +75,9 @@ namespace
 
             e.clsid = clsidMaybe;
 
-            json::object &val = prop.second.as_object();
-            e.assembly = val.at(_X("assembly")).as_string();
-            e.type = val.at(_X("type")).as_string();
+            auto &val = prop.value.GetObject();
```

Comment:
I am a huge fan of `auto` when the type is observed on the right side _or_ when using iterators. In this case, the type is opaque from usage and difficult to determine in review. I personally am okay with this but other might have a different opinion, just an FYI.

Reply:
Personally, I'm a fan of the explicit type.